action.yaml

```
@@ -0,0 +1,96 @@
+---
+name: "Presidio client - execute analyze action"
+description: "Github Action to execute Presidio framework using presidio_cli"
+inputs:
+  verify-path:
+    description: Path to verify
+    required: false
+    default: "."
+  configuration-file:
+    description: Path to configuration file or predefined configuration (default, limited)
+    required: false
+    default: "default"
+  configuration-data:
+    description: Simple configuration data directly
+    required: false
+    default: ""
+  format-output:
+    description: Format of output
+    required: false
+    default: "auto"
+  publish:
+    description: Publish result in PR comment
+    required: false
+    default: "true"
+  upload:
+    description: Uploads as artifact
+    required: false
+    default: "true"
+  presidio-cli-version:
+    description: Presidio cli version
```

```suggestion
    description: Presidio CLI version
```

Fun fact - CLI is not _client_, but _Command Line Interface_